src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapping.java

```diff
@@ -17,12 +17,16 @@
 public class InputNoLineWrapping {
 
   public void fooMethod() {
-      //
+    //
   }
 
-  // Long line ----------------------------------------------------------------------------------------
+  // Long line
+  // --------------------------------------------------------------------------------------------------
   // violation above 'Line is longer than 100 characters (found 103).'
 
-  private int[] mInts = new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, };
-  // violation above 'Line is longer than 100 characters (found 104).'
+  private int[] mInts =
+      new int[] {
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
+      };
```

Comment:
had to add extra numbers to make the line exceed column limit to test the section properly, but as mentioned here: https://github.com/checkstyle/checkstyle/pull/15280#discussion_r1678238140 formatter isn't letting me do it.

Reply:
https://checkstyle.org/google_style.html#a3.3.2

Reply:
>No line-wrapping
Import statements are not line-wrapped. The column limit (Section 4.4, [Column limit: 100](https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.4-column-limit)) does not apply to import statements.

I am ok to loose this violation. It is irrelevant to chapter.